### PR TITLE
Advanced search logic daniel

### PR DESCRIPTION
**Pull request by Daniel-Watkins**
I added a controller called SearchContoller and a viewmodel called SearchResultsViewModel. There is a method in SearchController called SearchResults that handles what athletes to display for the search results view. Search controller still needs a get and post for the Advanced search view. This pull request is primarily for review before the progress report on 4/15.

### CODE REVIEW

**Review by ksdurg — APPROVED** [review]
Looks good, Daniel.

**Review by AlexKSparks — COMMENTED** [review]
Looks good!

**Review by AlexKSparks — APPROVED** [review]

File: gybitg/Models/SearchViewModels/SearchResultsViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using gybitg.Models;
+
+namespace gybitg.Models.SearchViewModels
+{
+    public class SearchResultsViewModel
+    {
+        [Key]
+        public string UserId { get; set; }
+
+        [Display(Name = "First Name")]
```
Comment (AlexKSparks) [inline gybitg/Models/SearchViewModels/SearchResultsViewModel.cs]: Do we need first and last name in the viewmodel? Why not just have a get; set; on FullName as your controller just takes a string of a full name anyways?

**Review by adamkohler — APPROVED** [review]
It all looks good.  Since you went to the trouble of creating a new VM I think that we could use that same VM in the search post methods and pass that as the parameter to your advanced search logic method.  That will also depend though on what Kevin and Alex have done in the search views and what we decide to do in regards to the post methods from those views.

**Review by keithrutler — APPROVED** [review]
Looks good to me so far

**Review by AlexKSparks — APPROVED** [review]

**Pull request merged**